DOC-4318: Document the new feature of Allowing users to classify upla… (#128)
exoplatform/exo-documentation: docs/Manage-Documents.rst

   Cliking on upload manually opens a window showing a drive from your
   hard disk (it opens files drive for mobile devices) and allows you to
   select one or many files.

>    Same as for dropped files, you can select a destination folder for each file

Reviewer: > Same as for dropped files, you can select a destination folder

You can select a destination folder for each uploaded file on clicking ...

   file is already attached.

   |image167|
   
.. _AS_Files_Classification:   

Activity stream file's classification
````````````````````````````````````````

When sharing multiple files in the activity stream by uploading them from your hard disk
either by dragging and dropping or uploading them manually, you can classify them and choose
a destination folder in eXo Documents to store them.

For that purpose, you just need to click on ``Select destination folder`` link.

|dest_folder|

A popup of the documents drives appears allowing you to select a destination folder.

> .. note:: When uploading files from a space, the popup list folders of the space's drive, otherwise, it's your **Personal Documents**.

Reviewer: The default destination folder changes switch displayed activity stream:
* Space activity stream: the space home folder is displayed
*  Home activity stream: the user personal folder is displayed.

   file is already attached.

   |image167|
   
.. _AS_Files_Classification:   

Activity stream file's classification
````````````````````````````````````````

When sharing multiple files in the activity stream by uploading them from your hard disk
either by dragging and dropping or uploading them manually, you can classify them and choose

> a destination folder of your choice to store them.

Reviewer 2: ```suggestion
a destination folder in eXo Documents to store them.
```

For that purpose, you just need to click on ``Select destination folder`` link.

|dest_folder|

A popup of the documents drives appears allowing you to select a destination folder.

> .. note:: If you are uploading files in your activity stream, the select folder popup opens in 

Reviewer 2: ```suggestion
.. note:: When uploading files from a space, the popup list folders of the space's drive, otherwise, it's your **Personal Documents**.
```

>           **Personal Documents** drive.

Reviewer 2: ```suggestion
```

>           If you are uploading files in a space's activity stream, the select folder popup opens in 

Reviewer 2: ```suggestion
```

>           space's drive.

Reviewer 2: ```suggestion
```

                    
After selecting the files to upload from your hard disk, you can define a destination folder for each of them

> by clicking on |folder_icon| in the file's line.

Reviewer 2: ```suggestion
by clicking on |folder_icon| at the end of the file's row.
```

 
|destination_file|

> After choosing a destination folder, the select folder button switch to display the drive selected with a cancale icon

Reviewer 2: ```suggestion
After choosing a destination folder, the Select Folder button switches to display the destination folder with a cancel icon
```

> allowing to restore the default choices.

Reviewer 2: ```suggestion
allowing to restore the default destination.
```

|destination_selected|

> -  If the user cancels a destination folder for a file, the default destination will be the choice done for all files

Reviewer: -  If the user deletes the selected destination folder for a file (using cross button), the default one will be used

> -  If the user cancels a destination folder for a file, the default destination will be the choice done for all files

Reviewer 2: ```suggestion
-  If you cancel a destination folder for a file, the default destination will be applied to all files
```

> -  If the user cancels the selected destination folder for all files, files will be stored in the default folder i.e. 

Reviewer: -  If the user deletes the selected default destination folder for all files, the folder ``Activity Stream Documents`` will be used.

> -  If the user cancels the selected destination folder for all files, files will be stored in the default folder i.e. 

Reviewer 2: ```suggestion
-  If you cancel the selected destination folder for all files, they will be stored in the default folder i.e. 
```

>    ``Activity Stream Documents`` if no selected destination for them.

Reviewer 2: ```suggestion
   ``Activity Stream Documents``.
```
